lib/service_check.pm

```diff
@@ -140,15 +165,44 @@ sub install_services {
 
 sub check_services {
     my $service = shift;
```

Comment:
I would change this to `my ($service) = @_;` as you are doing below in apache_service_check().

Reply:
done.